src/panels/lovelace/editor/config-elements/hui-logbook-card-editor.ts

```diff
@@ -60,8 +64,16 @@ export class HuiLogbookCardEditor
     this._config = config;
   }
 
-  get _entities(): string[] {
-    return this._config!.entities || [];
+  get _targetPicker(): HassServiceTarget {
+    const entities = this._config!.entities || [];
+    if (this._config!.entities) {
+      delete this._config!.entities;
+    }
```

Comment:
Have you tested this?
I get `TypeError: Cannot delete property 'entities' of #<Object>`

Reply:
Definitely tested, back and forth. I'll take a look.